bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -447,16 +448,22 @@ void PreProcessor::onMessage<ClientBatchRequestMsg>(ClientBatchRequestMsg *msg)
                 << KVLOG(senderId, clientId, cid, clientBatch->numOfMessagesInBatch()));
   if (!checkClientBatchMsgCorrectness(clientBatch)) {
     preProcessorMetrics_.preProcReqIgnored.Get().Inc();
+    LOG_ERROR(logger(),
+              "ClientBatchPreProcessRequestMsg failed correctness check"
+                  << KVLOG(senderId, clientId, cid, clientBatch->numOfMessagesInBatch()));
```

Comment:
There are a lot of cases in which checkClientMsgCorrectness fails and they have specific logs. Not all of them are errors. Please remove this log as redundant.